platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -0,0 +1,346 @@
+
+Dependencies on the TSS-Roster
+
+* The `Roster` API is defined with an AddressBook wrapper for initial development and testing.
+    * The Roster API must be expanded to include the number of shares each node is assigned.
+* A lot can be done in parallel, but final integration will require that the TSS-Roster proposal be fully implemented.
+    * Transferring the ledger key will require the new roster life-cycle be complete and working.
+
+Impacts to Services Team
+
+* The app will need to specify how many shares are assigned to each node when it sets a new consensus roster.
+* Services will need to invoke the new ledger signing API to sign a block.
+    * This API is asynchronous and will return a future that returns the ledger signature when it is ready.
+* The HAPI transactions for add/update of consensus node in the address book must support a required EC public key.
+
+Impacts to DevOps Team
+
+* Each consensus node will need a new private long term EC key in addition to the existing RSA key.
+    * EC Key generation will have to happen before a node can join the network.
+* A new node added to an existing network will need to be given a state from an existing node after the network has
+  adopted the consensus roster containing the new node.
+
+Implications Of Completion
+
+* The consensus nodes will be able to create ledger signatures for use in block proofs.
+
+### Requirements
+
+TSS Core Requirements
+
+| Requirement ID | Requirement Description                                                                                                      |
+|----------------|------------------------------------------------------------------------------------------------------------------------------|
+| TSS-001        | The ledger MUST have a public/private signing key pair that does not change when the consensus roster changes.               |
+| TSS-002        | The ledger private key MUST be a secret that nobody knows.                                                                   |
+| TSS-003        | The `ledger id` MUST be a public key that everyone knows and able to verify signatures from the ledger private key.          |
+| TSS-004        | The ledger signature for a message MUST be produced through a threshold number of consensus nodes signing the message.       |
+| TSS-005        | There MUST be a way to bootstrap an existing network's consensus roster with to give the ledger a public/private signing key |
+| TSS-006        | There MUST be a way for a new network to generate a public/private signing key for the ledger.                               |
+| TSS-007        | There MUST be a way to rotate the ledger's public/private signing key.                                                       |
+| TSS-008        | The ledger signature SHOULD be verifiable by an EVM smart contract without excessive cost or time.                           |
```

Comment:
I do not understand how the EVM and smart contracts fit into that topic. Later in the proposal you say

> Our first implementation of Groth21 will use the ALT_BN128 elliptic curve which is in use and verifiable by EVMs.

That is all I can find regarind EVM usage / smart contracts. Can you please add somewhere an information how and why it is used by the EVM.